tests/tools/sgx-lkl-docker/README.md

```diff
@@ -0,0 +1,11 @@
+This test does not run SGX-LKL on the host but inside a Docker container.
+For this, a self-contained SGX-LKL installation is mounted into the container.
+To run this test, make sure you installed SGX-LKL and made it self-contained:
+
+```sh
+export SGXLKL_PREFIX=/opt/sgx-lkl
+sudo make install PREFIX=$SGXLKL_PREFIX
+sudo -E scripts/make_self_contained.sh
```

Comment:
Out of curiosity, why isn't this done automatically when SGX-LKL is installed?

Reply:
Because this is just one particular distribution/packaging mechanism optimized for mount-into-Docker usage. I can imagine that people may want to use SGX-LKL in different ways than that as well. I agree it should be integrated into the makefiles but maybe as separate install target, like `install-contained`.

Reply:
Is there harm in having this as the default?

Reply:
Apart from being a little unusual I don't think so. I'd prefer to do this in another PR though as it involves a few changes in the build system.